packages/mdc-dialog/index.js

```diff
@@ -171,7 +172,10 @@ class MDCDialog extends MDCComponent {
       releaseFocus: () => this.focusTrap_.deactivate(),
       isContentScrollable: () => !!this.content_ && util.isScrollable(/** @type {!Element} */ (this.content_)),
       areButtonsStacked: () => util.areTopsMisaligned(this.buttons_),
-      getActionFromEvent: (event) => event.target.getAttribute(strings.ACTION_ATTRIBUTE),
+      getActionFromEvent: (event) => {
```

Comment:
Need documentation for `getActionFromEvent` in dialog's README.

Reply:
Thanks, I forgot to update the wording there.